Mathlib/Algebra/DirectLimit.lean

```diff
@@ -269,7 +271,7 @@ theorem totalize_of_not_le {i j} (h : ¬i ≤ j) : totalize G f i j = 0 :=
 
 end Totalize
 
-variable [DirectedSystem G fun i j h => f i j h]
+variable [DecidableEq ι] [DirectedSystem G fun i j h => f i j h] {G f}
```

Comment:
```suggestion
variable [DecidableEq ι] [DirectedSystem G fun i j h => f i j h] 
variable {G f}
```
We should always separate binder updates from new variables. Eventually I'd like a linter enforcing this.